playwright.config.ts

```diff
@@ -4,7 +4,7 @@ import {devices} from "@playwright/test"
 const config: PlaywrightTestConfig = {
   testDir: "./e2e",
   /* Maximum time one test can run for. */
-  timeout: 50 * 1000,
+  timeout: 30 * 1000,
```

Comment:
for some reason this annoys me slightly - having to do a calculation just to, well, replace a comma .. could we use this form instead:
```timeout: 30_000```

Reply:
💯 .  I prevented to change the syntax as i've seen this being used in some playwright literature and maybe it is considered best practice but I prefer in ms.  Will change it

Reply:
convention is good, but this is basic js/ts, we don't need Playwright to guide is in that respect 😁